site/content/docs/0.0/layout/breakpoints.md

```diff
@@ -11,4 +11,188 @@ aliases:
+
+- **Use media queries to architect your CSS by breakpoint.** Media queries are a feature of CSS that allow you to conditionally apply styles based on a set of browser and operating system parameters. We most commonly use `min-width` in our media queries.
+
+- **Mobile first, responsive design is the goal.** OUDS Web's CSS aims to apply the bare minimum of styles to make a layout work at the smallest breakpoint, and then layers on styles to adjust that design for larger devices. This optimizes your CSS, improves rendering time, and provides a great experience for your visitors.
+
+## Available breakpoints
+
+OUDS Web includes eight default breakpoints, sometimes referred to as _grid tiers_, for building responsively. These breakpoints can be customized if you're using our source Sass files.
+
+{{< bs-table "table" >}}
+| Breakpoint | Class infix | Dimensions |
+| --- | --- | --- |
+| 2X-small | <em>None</em> |&lt;390px |
+| X-small | `xs` | &ge;390px |
+| Small | `sm` | &ge;480px |
+| Medium | `md` | &ge;736px |
+| Large | `lg` | &ge;1024px |
+| X-large | `xl` | &ge;1320px |
+| 2X-large | `2xl` | &ge;1640px |
+| 3X-large | `3xl` | &ge;1880px |
+{{< /bs-table >}}
+
+Breakpoints are representative of a subset of common device sizes and viewport dimensions—they don't specifically target every use case or device. Instead, the ranges provide a strong and consistent foundation to build on for nearly any device.
+
+These breakpoints are customizable via Sass—you'll find them in a Sass map in our `_variables.scss` stylesheet.
+
+{{< scss-docs name="grid-breakpoints" file="scss/_variables.scss" >}}
+
+For more information and examples on how to modify our Sass maps and variables, please refer to [the CSS section of the Grid documentation]({{< docsref "/layout/grid#css" >}}).
+
+## Media queries
+
+Since OUDS Web is developed to be mobile first, we use a handful of [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries) to create sensible breakpoints for our layouts and interfaces. These breakpoints are mostly based on minimum viewport widths and allow us to scale up elements as the viewport changes.
+
+### Min-width
+
+OUDS Web primarily uses the following media query ranges—or breakpoints—in our source Sass files for our layout, grid system, and components.
+
+```scss
+// Source mixins
+
+// No media query necessary for 2xs breakpoint as it's effectively `@media (min-width: 0) { ... }`
+@include media-breakpoint-up(xs) { ... }
+@include media-breakpoint-up(sm) { ... }
+@include media-breakpoint-up(md) { ... }
+@include media-breakpoint-up(lg) { ... }
+@include media-breakpoint-up(xl) { ... }
+@include media-breakpoint-up(2xl) { ... }
+@include media-breakpoint-up(3xl) { ... }
+
+// Usage
+
+// Example: Hide starting at `min-width: 0`, and then show at the `sm` breakpoint
```

Comment:
They use `sm` in Bootstrap because it's the first one. We could use `xs` instead here.